scripts/quests/otherAreas/Test_My_Mettle.lua

```diff
@@ -0,0 +1,438 @@
+    local expiry = quest:getVar(player, 'expiry')
+
+    -- expiry of 0 indicates either the quest has never been started,
+    -- previously successful, or a previous failure, not ignorance
+    if expiry == 0 then
+        local noSandalsAnywhere = not player:hasItem(xi.items.POWER_SANDALS)
+        local repeatable = canRepeat(player)
+
+        -- however, the sandals could have been in the player's storage
+        if noSandalsAnywhere and repeatable then
+            return quest:progressEvent(npcEventID.devean.OFFER_QUEST, 0, xi.items.POWER_SANDALS)
+        end
+
+        if trade ~= nil then
+            -- if the sandals were traded, Devean says too bad
+            return quest:progressEvent(npcEventID.devean.FAILURE)
+        end
+
+        -- player must throw away the sandals, otherwise they could try to
+        -- flag the quest after a failed repeat period, and then
+        -- immediately trade in
+        return quest:event(npcEventID.devean.DEFAULT)
+    end
+
+    -- expiry was set, so continue normally
+    local timeRemaining = expiry - os.time()
+   
+    if timeRemaining < 0 then
+        timeRemaining = 0
+    end
+
+    -- have to check for sandals in case they were not traded
+    local noSandalsInInventory = not player:hasItem(xi.items.POWER_SANDALS, xi.inv.INVENTORY)
+
+    if timeRemaining > 0 then
+        if noSandalsInInventory then
+            local gameHours = math.floor(timeRemaining / 150)
+
+            -- don't report 0 game hours remaining, we have already established
+            -- that the time remaining is greater than zero
+            if gameHours == 0 then
+                gameHours = 1
+            end
+
+            return quest:event(npcEventID.devean.GAME_HOURS_REMAINING, xi.items.POWER_SANDALS, gameHours)          
+        end
+
+        return quest:progressEvent(npcEventID.devean.SUCCESSFUL)
+    end
+
+    -- times up
+    if noSandalsInInventory then
+        return quest:progressEvent(npcEventID.devean.FAILURE)
+    else
+        -- cannot trigger trade unless it has only the sandals, so don't need
+        -- to test if the trade, exc
+        if trade == nil then
+            -- if the sandals were not traded, Devean says to toss or trade
+            return quest:event(npcEventID.devean.FAIL_HAS_SANDALS, xi.items.POWER_SANDALS)
+        end
+
+        -- if the sandals were traded, Devean says too bad
+        return quest:progressEvent(npcEventID.devean.FAILURE)
+    end
+end
+
+local function recordOption(player, option)
+    -- option examples:
+    -- option = 1073741824 value for first, second, third, and fourth decline
+    -- binary = 0100 0000 0000 0000 0000 0000 0000 0000
+    -- option = 124  100 gil  24 hours
+    -- option = 286  250 gil  36 hours
+    -- option = 322  250 gil  72 hours
+    -- option = 548  500 gil  48 hours
+    -- option = 1060 1000 gil 60 hours
+    -- option = 1072 1000 gil 72 hours
+    
+    if option ~= 1073741824 then
```

Comment:
This value is used in a few other cancelled dialog scenarios, such as Lest We Forget, Rune of Transfer, and Splintery Chest.  Can you reference an example, so that I can follow a better example?

Reply:
@claywar I found a potentially better example in `besieged.lua` and `atma.lua` that uses `option < 0x40000000`.  Perhaps I can open another PR after this one to standardize the usage.  What is your take?